docs/For User/Basic Concepts/02 Tokens.md

```diff
@@ -0,0 +1,22 @@
+# Tokens
+
+## Introduction
+IMV is a Cosmos-based chain with full Ethereum Virtual Machine (EVM) support. Because of this architecture, tokens and assets in the network may come from different independent sources.
+
+## The IMV Token
+The denomination used for staking, governance and gas consumption on the EVM is the IMV. The IMV provides the utility of: securing the Proof-of-Stake chain, token used for governance proposals, distribution of fees to validator and users, and as a mean of gas for running smart contracts on the EVM.
+
+IMV uses [Atto](https://en.wikipedia.org/wiki/Atto-) IMV as the base denomination to maintain parity with Ethereum.
+
+* 1 imv = 1018 aimv

```

Comment:
Change 1 imv = 1018 aimv
to 1 imv = 10^18 aimv

Reply:
Updated